lib/tasks/push-to-space/creation.ts

```diff
@@ -98,23 +99,23 @@ async function createEntitiesInSequence ({ context, entities, destinationEntitie
 /**
  * Creates a list of entries
  */
-export async function createEntries ({ context, entities, destinationEntitiesById, requestQueue }) {
+export async function createEntries ({ context, entities, destinationEntitiesById, skipUpdates, requestQueue }) {
   const createdEntries = await Promise.all(entities.map((entry) => {
-    return createEntry({ entry, target: context.target, skipContentModel: context.skipContentModel, destinationEntitiesById, requestQueue })
+    return createEntry({ entry, target: context.target, skipContentModel: context.skipContentModel, destinationEntitiesById, skipUpdates, requestQueue })
   }))
 
   return createdEntries.filter((entry) => entry)
 }
 
-async function createEntry ({ entry, target, skipContentModel, destinationEntitiesById, requestQueue }) {
+async function createEntry ({ entry, target, skipContentModel, destinationEntitiesById, skipUpdates, requestQueue }) {
   const contentTypeId = entry.original.sys.contentType.sys.id
   const destinationEntry = getDestinationEntityForSourceEntity(
     destinationEntitiesById, entry.transformed)
-  const operation = destinationEntry ? 'update' : 'create'
+  const operation = destinationEntry ? (skipUpdates ? 'skip' : 'update') : 'create'
```

Comment:
Just a personal preference here but I find nested ternaries to be less readable than a switch or if else

Reply:
Thanks for the review @david-shibley-contentful! Unrolled the former into two expressions, substituted the latter with an if-else statement

Reply:
(missing having if-else as an expression!)